jawt-docs/dev/process.md

```
@@ -0,0 +1,256 @@
+
+## Key Data Structures
+
+### `ProcessStatus`
+
+An enumeration for the different states a managed process can be in.
+
+```go
+type ProcessStatus int
+
+const (
+	StatusStopped ProcessStatus = iota
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)